src/main/java/frc/robot/commands/ArmToAngleCmd.java

```diff
@@ -3,26 +3,26 @@
 import dev.doglog.DogLog;
 import edu.wpi.first.wpilibj2.command.Command;
 import frc.robot.subsystems.ArmSubsystem;
-import java.util.function.Supplier;
+import java.util.function.DoubleSupplier;
 
 public class ArmToAngleCmd extends Command {
-  public ArmToAngleCmd(Supplier<Double> angle, ArmSubsystem arm) {
+  public ArmToAngleCmd(DoubleSupplier angle, ArmSubsystem arm) {
```

Comment:
More `Supplier<Double>` -> `DoubleSupplier` changes, see previous comment for more information 🙂